src/javascript/JContent/ContentRoute/ContentPath/ContentPath.jsx

```diff
@@ -50,11 +50,11 @@ const renderItems = (items, onItemClick) => {
         return [
             <SimplePathEntry key={first.uuid} item={first} onItemClick={onItemClick}/>,
             <CompositePathEntry key={`${first.uuid}-${last.uuid}`} items={others} onItemClick={onItemClick}/>,
-            <SimplePathEntry key={last.uuid} item={last} onItemClick={onItemClick}/>
+            <SimplePathEntry key={last.uuid} item={last}/>
         ];
     }
 
-    return items.map(item => <SimplePathEntry key={item.uuid} item={item} onItemClick={onItemClick}/>);
+    return items.map((item, index) => <SimplePathEntry key={item.uuid} item={item} {...(index === items.length - 1 ? {} : {onItemClick: onItemClick})}/>);
```

Comment:
`onItemClick={index === items.length - 1 ? undefined : onItemClick}` maybe?
The unnecessary unpacking looks a bit hacky